src/js/intercom.ts

```diff
@@ -0,0 +1,35 @@
+export function login(email: string) {
+  const intercom = (window as any).Intercom;
+  if (intercom) {
+    intercom('boot', {
+      app_id: 'i2twhziy',
```

Comment:
It's our app_id from Intercom. We should move this to an environment variable. The Intercom stuff is very MVP right now, but I'd prefer to refactor and improve this (and the event type constants) in a separate PR.